pkg/actor/decommission.go

```diff
@@ -85,13 +85,6 @@ func (d decommission) Act(ctx context.Context, cluster *resource.Cluster) error
 	}
 
 	nodes := uint(cluster.Spec().Nodes)
-	//We do not scale down if the nodes field is less than 3
-	//TODO @alina add validation webhook (see https://github.com/cockroachdb/cockroach-operator/issues/245)
-	if nodes < 3 {
-		err := errors.New("decommission with less than 3 nodes is not supported")
-		log.Error(err, "We cannot decommission if there are less than 3 nodes", "nodes", nodes)
```

Comment:
Any harm in leaving this in?

Reply:
Harm may be strong. My thinking here was that if we believe the admission controller works, we shouldn't need to duplicate validation logic throughout the code base. This one was done simply by adding a validation marker to the CRD, which is handled by the admission controller (even without our custom webhook).